SoojungKim/baekjoon/BOJ_11022.java

```diff
@@ -0,0 +1,39 @@
+package baekjoon;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class BOJ_11022 {
+
+    public static final Scanner scanner = new Scanner(System.in);
+    public static void main(String[] args) {
+
+        int n;
+        String str;
+
+        str = scanner.nextLine();
+        n = Integer.parseInt(str);
```

Comment:
```suggestion
        int n = Integer.parseInt(str);
```

리터럴 표기법이 더 가독성이 좋습니다 !